core/src/test/java/org/elasticsearch/search/aggregations/BasePipelineAggregationTestCase.java

```diff
@@ -81,7 +81,7 @@
 import static org.elasticsearch.cluster.service.ClusterServiceUtils.setState;
 import static org.hamcrest.Matchers.equalTo;
 
-public abstract class BasePipelineAggregationTestCase<AF extends PipelineAggregatorBuilder> extends ESTestCase {
+public abstract class BasePipelineAggregationTestCase<AF extends AbstractPipelineAggregatorBuilder<AF>> extends ESTestCase {
```

Comment:
Should this be `BasePipelineAggregationTestCase<AB extends PipelineAggregatorBuilder>`